### ISSUE TITLE
Code suggestion

### ISSUE THREAD

**Issue by RKDUDE02** [phpmaple/Stick-Hero-Swift#6]
I notice that in GameViewController.swift that when it tries to play a sound it prints NO AUDIO PLAYER. Instead maybe we could use a UIAlert Controller telling the user that sound is not working.

**Comment by phpmaple** [comment]
> I notice that in GameViewController.swift that when it tries to play a sound it prints NO AUDIO PLAYER. Instead maybe we could use a UIAlert Controller telling the user that sound is not working.

Yep, right! We should fully-tested In pre production, and If audio files not exist we should send error message to our server instead of alert message to user, that will be better for user, this program just demonstrate, very good suggestion!

**Comment by RKDUDE02** [comment]
> > I notice that in GameViewController.swift that when it tries to play a sound it prints NO AUDIO PLAYER. Instead maybe we could use a UIAlert Controller telling the user that sound is not working.
> 
> Yep, right! We should fully-tested In pre production, and If audio files not exist we should send error message to our server instead of alert message to user, that will be better for user, this program just demonstrate, very good suggestion!

That's even better so that we can handle it. But quick question, do we already have servers or do we need to add some? And if we need to add some i suggest that we use firebase.
